packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -678,6 +683,220 @@ function wait(time) {
   });
 }
 
+function handleErrorMsg(err) {
+  cliux.print(`Error: ${err?.errorMessage ? err?.message: messageHandler.parse('CLI_EXPORT_CSV_API_FAILED')}`, { color: 'red' });
```

Comment:
@cs-raj Can you recheck this condition once again?

This seems wrong code: `err?.errorMessage ? err?.message : ''`


I suppose it should look as follows: `cliux.print('Error: ${(err?.errorMessage || err?.message) ? err?.errorMessage || err?.message : messageHandler.parse('CLI_EXPORT_CSV_API_FAILED')}', { color: 'red' })`

Reply:
Corrected it @antonyagustine